### PR TITLE
fixes #5414: LDAP + TLS is not LDAP + TLS but LDAP + SSL

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

**Comment by orrabin** [comment]
Do not merge since I would like to auto change the port

**Comment by isratrade** [comment]
@orrabin, looks good to me.

File: app/assets/javascripts/auth_source_ldap.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function change_ldap_port(item) {
+  var port = $('#auth_source_ldap_port');
+  if ($(item).is(':checked')) {
+    if (port.val() == port.data('default-ports')['ldap']) {
```
Comment (ohadlevy) [inline app/assets/javascripts/auth_source_ldap.js]: while not critical, could we do some basic validations first (or at least some exception handling? e.g. if

data default port is not properly defined
if port var is null
etc ?

File: app/assets/javascripts/auth_source_ldap.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function change_ldap_port(item) {
+  var port = $('#auth_source_ldap_port');
+  if (port.length > 0) {
+    var value = port.val();
```
Comment (dLobatog) [inline app/assets/javascripts/auth_source_ldap.js]: Could you rename this to port_value? Just 'value' seems very vague.

**Comment by dLobatog** [comment]
:+1: , I'll merge after that nitpick, sorry...

**Comment by orrabin** [comment]
@eLobato changed it

**Comment by domcleal** [comment]
Rather than changing the s_() (column translation) and labels, should the column be renamed so the API is also consistent?

**Comment by ohadlevy** [comment]
@domcleal I'm in favor of updating the DB/API, but IMHO in a second PR, as we would need to keep backward compatibility for API v1.

**Comment by dLobatog** [comment]
Merged as https://github.com/theforeman/foreman/commit/42163f861cfa9c5446abbe5d53035365085e441d , thanks @orrabin !

**Pull request closed without merging**